src/frontend/src/components/EditableTextInputList.tsx

```diff
@@ -88,14 +91,18 @@ const EditableTextInputList: React.FC<EditableTextInputListProps> = ({
   let listPrepared = items.map((item, index: number) =>
     !readOnly ? (
       <li key={index} className={'mb-2'}>
-        <InputGroup>
-          <Form.Control
+        <Box marginBottom={1}>
+          <TextField
             required
-            autoFocus={hasTyped && isLastElement(index)}
+            fullWidth
             type="text"
-            ref={isLastElement(index) ? focusRef : null}
+            id={`bullet-${index}`}
+            name={`bullet-${index}`}
             value={item.toString()}
-            placeholder={'Input new bullet here...'}
+            label="Work Package Name"
+            placeholder="Input new bullet here..."
+            ref={isLastElement(index) ? focusRef : null}
+            autoFocus={hasTyped && isLastElement(index)}
```

Comment:
Is the idea here that the last element in the list autofocuses? Or is this just to focus the last element in the list while you're typing in it?

Reply:
yep, I didn't code this, I think Anthony did.